### ISSUE TITLE
Compatible with Angular2 Final version ?

### ISSUE THREAD

**Issue by sireneweb** [softsimon/angular-2-dropdown-multiselect#15]
Hi, could you confirm it's compatible with Angular2 final version and New module implementation from RC7  with NgModule ?

**Comment by kalras** [comment]
Seems like it is compatible with Angular2 final version. I just tested it.

**Comment by kmckinley** [comment]
I haven't been able to get it to work in Angular2 final.   I used Angular-cli and webpack.  When I add it via NPM it can't find it to import into NgModule.  I have to manually add `import {MultiselectDropdown} from "angular-2-dropdown-multiselect/src/multiselect-dropdown";`  Maybe I'm doing something wrong?

Once I do that it can atleast find MultiselectDropdown, but when I try to actually use it I get an error `Unexpected directive 'MultiselectDropdown' imported by the module '*'`

@kalras You mind sharing how you got it to work?  

Thanks!

**Comment by kalras** [comment]
This is what I did:
Added following two items to my parent component's module.ts

import {MultiselectDropdownModule} from "angular-2-dropdown-multiselect/multiselect-dropdown";
and
@NgModule({
    imports: [MultiselectDropdownModule]

And used the tag <ss-multiselect-dropdown> in a child component.

Did you include this anywhere else as a directive: Search online shows the following:
Source: http://stackoverflow.com/questions/39268583/angular-2-directives-declared-yet-still-have-errors
imports: [] must not contain directives, only modules. As the error message says, it is not a module. Directives should go into declarations: [...] of exactly one module and then add this module to imports: [] of the modules where you want to use it.
